lib/shopify_app/controller.rb

```diff
@@ -3,16 +3,6 @@
 module ShopifyApp
   module Controller
     extend ActiveSupport::Concern
-
     include ShopifyApp::LoginProtection
-
-    def fullpage_redirect_to(url)
-      if ShopifyApp.configuration.embedded_app?
-        render inline: %Q(<script type="text/javascript">window.top.location.href = #{url.to_json};</script>)
-      else
-        redirect_to url
-      end
-    end
-
```

Comment:
that makes sense. once the changes are in, i'll ping you and we can both run through a :tophat:.